airbyte-integrations/connectors/source-quickbooks/source_quickbooks/manifest.yaml

```diff
@@ -1,211 +1,1871 @@
-version: "1.0.0"
+version: 1.0.0
```

Comment:
Let's bump this to 4.3.2 while we're here.

Reply:
We'll have to bump the cdk version too then in poetry, is that fine?

Reply:
Done, manifest version is 4.3.2 now :)